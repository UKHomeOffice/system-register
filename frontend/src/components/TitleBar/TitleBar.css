.title-bar-title {
    font-weight: bold;
    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
}

.top-nav > div {
    padding-bottom: 18px;
    padding-top: 4px;
}

.title-bar-login {
    position: absolute;
    right: 5px;
    top: 10px;
    line-height: 2.3333;
}
