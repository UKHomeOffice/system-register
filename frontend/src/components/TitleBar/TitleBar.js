import React from 'react'
import Auth from '../Auth/Auth'
import './TitleBar.css'
import TopNav, { asNavLinkAnchor } from '@govuk-react/top-nav';

const NavAnchor = asNavLinkAnchor('a');


const TitleBar = () => {

    return (
        <TopNav data-testid="title-bar-div" company="" serviceTitle="System Register" active={0}></TopNav>
        // <div data-testid="title-bar-div" className="title bottomMargin">
        //     <h2 className="title">System Register</h2>
        //     <Auth />
        // </div>
    )
}

export default TitleBar