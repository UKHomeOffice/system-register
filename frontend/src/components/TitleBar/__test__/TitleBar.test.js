import React from 'react';
import { render, cleanup } from '@testing-library/react';
import TitleBar from '../TitleBar';
import snapshotRenderer from 'react-test-renderer'

afterEach(cleanup)

describe('<Title />', () => {
  it('renders without crashing', () => {
    render(<TitleBar />)
  })

  it('renders title', () => {
    const { getByTestId } = render(<TitleBar />);
    const title = getByTestId('title-bar-div');
    expect(title).toHaveTextContent('System Register')
  });

  it('matches snapshot', () => {
    expect(snapshotRenderer.create(<TitleBar />))
      .toMatchSnapshot()
  })
})