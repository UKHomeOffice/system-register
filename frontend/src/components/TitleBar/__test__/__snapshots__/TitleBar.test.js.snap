// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> matches snapshot 1`] = `
Array [
  <div
    className="sc-bdfBwQ iBYTrY"
    color="#ffffff"
    data-testid="title-bar-div"
  >
    <div
      className="sc-gsTCUz heEpZ"
    >
      <div
        className="sc-eCssSg hMcpmq"
      >
        <div
          className="sc-hKgILt klPTWy"
        >
          
        </div>
      </div>
      <div
        className="sc-jSgupP bPwDQU"
      >
        <div
          className="sc-bqyKva eoKSDV"
        >
          System Register
        </div>
      </div>
    </div>
  </div>,
  <div
    className="sc-dlfnbm ivQkZE"
  />,
]
`;
